src/components/interface/AudioPlay.jsx

```diff
@@ -1,26 +1,78 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+
 import { withStyles } from '@material-ui/core/styles';
 import PlayCircleOutlineIcon from '@material-ui/icons/PlayCircleOutline';
+
 import styles from '../styles';
+import Button from '../interface/Button';
+import * as firebase from 'firebase';
 
 class AudioPlayer extends React.Component {
   constructor(props) {
     super(props);
 
     this.state = {
-      disabled: false
+      audioFile: undefined,
+      metadataLoaded: false
     };
   }
 
-  // TODO: handle audio playing file
+  loadAudio = () => {
```

Comment:
I would typically argue that these should be alphabeticized but I think in React, the lifecycle components are more important to keep at the top.

How would you feel about the following styling:

Let's move the react lifecycle components under the constructor alphabetically, then these arrow functions below that alphabetically. Finally the Render at the end?

Reply:
Would it be better to put the arrow functions at the top then all the lifecycle components below with `render` at the end? That way all the lifecycle components including `render` will be grouped together.